### PR TITLE
Update RunStatusSensorContext to crag

### PR DESCRIPTION
**Pull request by dpeng817**

### CODE REVIEW

File: python_modules/dagster/dagster/core/definitions/run_status_sensor_definition.py [hunk 1]
```diff
@@ -106,6 +111,10 @@ def for_run_failure(self):
             instance=self.instance,
         )
 
+    @property
+    def pipeline_run(self) -> PipelineRun:
```
Comment (yuhan) [inline python_modules/dagster/dagster/core/definitions/run_status_sensor_definition.py:115]: maybe throw a deprecation warning?

**Review by yuhan — COMMENTED** [review]
we also need to update the code snippets on the docs: https://docs.dagster.io/concepts/partitions-schedules-sensors/sensors#job-failure-sensor which says `context.pipeline_run`

**Review by yuhan — COMMENTED** [review]
from reading the failing test, seems like we also need to update these lines: https://github.com/dagster-io/dagster/blob/master/python_modules/dagster/dagster/core/definitions/run_status_sensor_definition.py#L93-L107

**Review by yuhan — DISMISSED** [review]
back to your q

**Review by yuhan — APPROVED** [review]

File: python_modules/dagster/dagster/core/definitions/run_status_sensor_definition.py (hunk 1 above)
Comment (prha) [reply python_modules/dagster/dagster/core/definitions/run_status_sensor_definition.py:117]: Is it weird to say `as of 0.13.0`?  We'd already be at `0.13.10`

**Pull request merged**